core/app/models/spree/product/scopes.rb

```diff
@@ -28,12 +28,14 @@ def self.property_conditions(property)
           scope :ascend_by_name, -> { order(name: :asc) }
           scope :descend_by_name, -> { order(name: :desc) }
 
-          add_search_scope :ascend_by_master_price do
-            joins(master: :default_price).order(Spree::Price.arel_table[:amount].asc)
+          Spree::Product.add_search_scope :ascend_by_master_price do
```

Comment:
Is `Spree::Product.` necessary here?

Reply:
Nope 😄  It's just a monkey copy/paste by my decorator 😅